src/dmd/mars.d

```diff
@@ -1573,12 +1560,6 @@ private bool parseCommandLine(const ref Strings arguments, const size_t argc, re
                 params.multiobj = true;
             else if (arg == "-g") // https://dlang.org/dmd.html#switch-g
                 params.symdebug = 1;
-            else if (arg == "-gc")  // https://dlang.org/dmd.html#switch-gc
-            {
-                Loc loc;
-                deprecation(loc, "use -g instead of -gc");
-                params.symdebug = 2;
```

Comment:
I always wondered whether this deprecation is desirable. -gc is still used by the COFF backend in case you don't want/cannot use mago with the Visual Studio debugger.